static/app/utils/replays/replayerStepper.tsx

```diff
@@ -47,6 +47,8 @@ export default function replayerStepper<
 
     const onDone = () => {
       resolve(collection);
+      replayer.off('pause', handlePause);
```

Comment:
to avoid recursion, since `destroy()` calls `pause()`

Reply:
can you add this as a code comment?